doc/FAQ.md

```diff
@@ -34,13 +34,21 @@ distributed systems. I just wanted the fastest way to boot to Docker.
 
 **Persistent partition choice**
 
-Boot2Docker will first try to mount a partition labeled ``boot2docker-data``, if
+Boot2Docker will first try to mount a partition named ``boot2docker-data``, if
 that doesn't exist, it will pick the first ``ext4`` partition listed by ``blkid``.
 
+By default, the ISO image creates the ``boot2docker-vm`` partition, therefore no additional one is likely to be needed but you can still add some if you want to.
+
+Keep in mind that if you have 2 partitions with the same names but different labels,
+boot2docker will only see one of them. So, if you add a ``boot2docker-data`` partition, boot2docker will mount that one and forget ``boot2docker-vm``.
+However, if you name them differently, boot2docker will see all of them.
+E.g.: ``boot2docker-vm`` and ``partition2`` will both be mounted.
```

Comment:
Let's take `boot2docker-vm` as example. `boot2docker` would be the name part of it and `-vm` the label part. If we have 4 different data disk attached, respectively called `boot2docker-data`, `boot2docker-alpha`, `testname-one` and `testname-two`, boot2docker will attach `boot2docker-data`(because it is the first name it is looking for) then it will attach `testname-one` and will forget the two others because they have the same name.

Well, I didn't check this behaviour for days so I might be a little bit wrong. At least, I'm sure that `boot2docker-alpha` will be forgotten.

My above explanation should probably be in the doc, as it is a bit clearer I guess.

Feel free to change it as you wish to make it clearer, the English isn't my mother language.